libp2p/RLPXFrameCoder.h

```diff
@@ -33,18 +33,24 @@ namespace dev
 namespace p2p
 {
 
+struct RLPXFrameDecrytFailed: virtual dev::Exception {};
+
+/**
+ * @brief Encapsulation of Frame
+ * @todo coder integration; padding derived from coder
+ */
 struct RLPXFrameInfo
 {
 	RLPXFrameInfo() = default;
 	/// Constructor. frame-size || protocol-type, [sequence-id[, total-packet-size]]
 	RLPXFrameInfo(bytesConstRef _frameHeader);
-	uint32_t length = 0;			///< Max: 2**24
-	uint8_t padding = 0;
+	uint32_t length = 0;			///< Size of frame (excludes padding). Max: 2**24
+	uint8_t padding = 0;			///< Length of padding which follows @length.
 	
-	uint16_t protocolId = 0;
-	bool hasSequence = false;
-	uint16_t sequenceId = 0;
-	uint32_t totalLength = 0;
+	uint16_t protocolId = 0;		///< Protocol ID as negotiated by handshake.
+	bool multiFrame = false;		///< If this frame is part of a sequence
+	uint16_t sequenceId = 0;		///< Sequence ID of frame
+	uint32_t totalLength = 0;		///< Set to total length of packet in first frame of multiframe packet
```

Comment:
Can you make some of the fields (possibly all of them) const? That would make clear that they are set when FrameInfo is created and never modified.